core/capabilities/launcher.go

```diff
@@ -381,7 +381,7 @@ func (w *launcher) addRemoteCapabilities(ctx context.Context, myDON registrysync
 
 		capabilityConfig, err := c.Unmarshal()
 		if err != nil {
-			return fmt.Errorf("could not unmarshal capability config for id %s: %w", cid, err)
+			return fmt.Errorf("could not unmarshal capability config for id %s with bytes: %s: %w", cid, hex.EncodeToString(c.Config), err)
```

Comment:
There is a formatting verb for this too 🤷 
```suggestion
			return fmt.Errorf("could not unmarshal capability config for id %s with bytes: %x: %w", cid, c.Config, err)
```
```
%x	base 16, lower-case, two characters per byte
%X	base 16, upper-case, two characters per byte
```